Fix missing variable reference in meta-box-partial-page.php

BE-Webdesign opened the pull request:
Fix minor PHP error. See
https://github.com/WordPress/gutenberg/pull/2804#discussion_r145954650

BE-Webdesign committed:
Fix missing variable reference in meta-box-partial-page.php

Fix minor PHP error. See
https://github.com/WordPress/gutenberg/pull/2804#discussion_r145954650

aduth commented on lib/meta-box-partial-page.php:
```diff
@@ -69,9 +69,12 @@ function gutenberg_meta_box_partial_page( $post_type, $meta_box_context ) {
 	global $post, $wp_meta_boxes, $hook_suffix, $current_screen, $wp_locale;
 
+	// This page should always match up with the edit action.
+	$action = 'edit';
```
If `$action` is always `'edit'` and `gutenberg_meta_box_partial_page_post_form` is only ever called from here, is the `'edit' === $action` even necessary? Are we expecting `gutenberg_meta_box_partial_page_post_form` to be called in other contexts?

BE-Webdesign replied:
Are we expecting gutenberg_meta_box_partial_page_post_form to be called in other contexts?

I do not know. We can most likely get rid of this code then? I am fairly positive that metaboxes only ever display if the edit action is set on post.php, but that could be wrong. In this code I am setting it to be edit anyways, so it is probably best to just eliminate any of the code using action.

BE-Webdesign replied:
Let me know if you want to merge as is, or remove any reference to action since it should always be edit anyways?

aduth replied:
>Let me know if you want to merge as is, or remove any reference to action since it should always be edit anyways.

I'm content with either, but if we don't anticipate to need to check `$action`, probably best to remove.

BE-Webdesign committed:
Remove $action entirely.

The $action variable is not relavent in this context of meta boxes, as
the $action should always be `edit`. We can assume that is the case. In
the future if other contexts are needed we can address those case by
case.
